src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -209,6 +209,16 @@ public void setScheduleBoard(ReadOnlyScheduleBoard scheduleBoard) {
         this.scheduleBoard.resetData(scheduleBoard);
     }
 
+    @Override
+    public Path getScheduleBoardFilePath() {
+        return userPrefs.getScheduleBoardFilePath();
+    }
+
+    @Override
+    public void setScheduleBoardFilePath(Path scheduleBoardFilePath) {
+        requireNonNull(scheduleBoardFilePath);
+        userPrefs.setScheduleBoardFilePath(scheduleBoardFilePath);
+    }
```

Comment:
```suggestion
    }
    
```
I think space after would be good here~